portal-impl/src/com/liferay/portal/service/impl/CountryLocalServiceImpl.java

```diff
@@ -124,6 +134,11 @@ public Country deleteCountry(long countryId) throws PortalException {
 		return countryLocalService.deleteCountry(country);
 	}
 
+	@Override
+	public <T> T dslQuery(DSLQuery dslQuery) {
+		return countryPersistence.dslQuery(dslQuery);
+	}
+
```

Comment:
This should be available by default in the base impl class. I don't think you need to override it here.